[FIX] broken link (#2883)
babel/website: website/data/tools/node/usage.md

<blockquote class="alert alert--info">
  <p>

>     For detailed insights and usage instructions on Babel Core, explore the <a href="/docs/babel-core">official Babel Core documentation</a>.

Reviewer: I think we can simplify this to just:

```suggestion
    Explore the <a href="/docs/babel-core">@babel/core documentation</a> for the full API.
```
